infra/base-images/base-builder-new/bazel_build_fuzz_tests

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash -eu
```

Comment:
Is this file supposed to be part of your change?

Reply:
So my reasoning is that this will eventually replace the base-builder-new... and given that bazel is language agnostic... I think it belongs in the base-builder-new?

Reply:
Sorry, this comment was confused. I understand what youre doing here.